homeassistant/components/ubus/device_tracker.py

```diff
@@ -110,6 +110,10 @@ def get_device_name(self, device):
         name = self.mac2name.get(device.upper(), None)
         return name
 
+    def get_extra_attributes(self, device):
```

Comment:
If you use the async method then there is no need for the executor job.
```suggestion
    async def async_get_extra_attributes(self, device: str) -> dict[str, str]:
```